Front-end/parquick/src/utils/testData.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import type { LatLngLiteral } from "../types/LatLngLiteral";
+
+export const generateParkings = (position: LatLngLiteral): Array<LatLngLiteral> => {
```

Comment:
Looks so nice (and safe!) with Flow!

You could probably even return `$ReadOnlyArray` if the result of this should be immutable.